contrib/checkbox-provider-ce-oem/units/optee/jobs.pxu

```diff
@@ -25,22 +25,13 @@ _summary: Check if xtest is in the gedget snap.
 plugin: shell
 user: root
 estimated_duration: 20.0
-environ: OPTEE_TOOL
 command:
-    if [[ -z "$OPTEE_TOOL" ]]; then
-        gadget=$(snap list | awk '$NF == "gadget" {print $1}')
-        xtest=$(snap info "$gadget"|grep xtest)
-        if [[ "$xtest" ]]; then
-            echo -e "\nInfo: Gadget snap support xtest command!"
-            echo -e "$xtest"
-            exit 0
-        else
-            echo -e "\nError: Gadget snap not support xtest command!"
-            echo -e "\nError: Please identify the tool name if needed!"
-            exit 1
-        fi
+    tool=$(parse_optee_test.py)
+    if [[ "$tool" != "None" ]]; then
```

Comment:
This test is a bit weird. It only checks if the string "None" is printed, but if the test fails, since whatever is printed in stderr is not equal to "None", the test will pass and output `echo "Info: Found xtest runnable $tool"`

Ideally, I would prefer avoiding bash scripting as much as possible, and handle everything in a python script, but at least add here a way to catch the errors on the script:
```suggestion
    exit_status=$?
    if [[ "$exit_status" -eq 0 ]]; then
        echo "Info: Found xtest runnable $tool"
```